pyproject.toml

```diff
@@ -21,10 +19,13 @@ ipytree = { version = "^0.2.2", optional = true }
 ipywidgets = { version = "^8.1.2", optional = true }
 traitlets = { version = "^5.14.3", optional = true }
 scipy = { version = "^1.14.1", optional = true }
+pandas = { version = "^2.2", optional = true }
+plotly = { version = "^5.17", optional = true }
 
 [tool.poetry.extras]
 jupyter = ["ipytree", "ipywidgets", "traitlets"]
 optimization = ["scipy"]
+visualization = ["pandas", "plotly"]
```

Comment:
@Brendan-Reid1991 How about having just one extra `interactive`, that would combine `jupyter` and `visualization`?
I understand that visualizations are not necessary part of the interactive workflow, but that's probably how 90% of people will be using it and it might be just better API.
WDYT?